app/src/main/java/ch/sdp/vibester/database/AppPreferences.kt

```diff
@@ -0,0 +1,40 @@
+package ch.sdp.vibester.database
+
+import android.content.Context
+import android.content.SharedPreferences
+
+object AppPreferences {
```

Comment:
consider adding documentation for this class

Reply:
added